Documentation/Input/HowToAddNearInteractivity.md

```diff
@@ -0,0 +1,148 @@
+
+```csharp
+public class PrintPointerEvents : MonoBehaviour, IMixedRealityPointerHandler
+{
+    public void OnPointerDown(MixedRealityPointerEventData eventData)
+    {
+        if (eventData.Pointer is SpherePointer)
+        {
+            Debug.Log($"Grab start from {eventData.Pointer.PointerName}");
+        }
+        if (eventData.Pointer is PokePointer)
+        {
+            Debug.Log($"Touch start from {eventData.Pointer.PointerName}");
+        }
+    }
+
+    public void OnPointerClicked(MixedRealityPointerEventData eventData) {}
+    public void OnPointerDragged(MixedRealityPointerEventData eventData) {}
+    public void OnPointerUp(MixedRealityPointerEventData eventData) {}
+}
+```
+
+## How to add near grab interactions
+1. Make sure you have a [SpherePointer](Pointers.md#spherepointer) in you MRTK pointer profile. If you are using the default MRTK profile or the default HoloLens 2 profile, you will have this already. If you have a custom profile, look under Input / Pointers  / Pointer Options. Make sure that for the "articulated hand" controller type you have an entry that points to the GrabPointer prefab under MRTK.SDK/Features/UX/Prefabs, or create your own prefab and add a [SpherePointer](Pointers.md#spherepointer).
+
+1. Pick an object that you want to make grabbable. Any ancestor of that object will be able to receive pointer events. For example, you can add a cube to the scene.
+
+1. Make sure there is a collider to that object.
+
+1. Make sure the layer your object is on is a grabbable layer. By default, all layers except Spatial Awareness and Ignore Raycast are grabbable. You can see which layers are grabbable by inspecting the Grab Layer Masks in your GrabPointer prefab.
+
+1. Add a [NearInteractionGrabbable](xref:Microsoft.MixedReality.Toolkit.Input.NearInteractionGrabbable) to that collider.
+
+1. On that object or one of its ancestors, add a component that implements [IMixedRealityPointerHandler](xref:Microsoft.MixedReality.Toolkit.Input.IMixedRealityPointerHandler). You can for example add [ManipulationHandler](xref:Microsoft.MixedReality.Toolkit.UI.ManipulationHandler).
+
+
+## How to add near touch interactions to collidables
+> If you have UnityUI that you would like to make work with touch events, please read the next section
+
+1. Make sure you have a [PokePointer](Pointers.md#pokepointer) in your MRTK pointer profile. If you are using the default MRTK profile or the default HoloLens 2 profile, you will have this already. If you have a custom profile, look under Input / Pointers  / Pointer Options. Make sure that for the "articulated hand" controller type you have an entry that points to the PokePointer prefab under MRTK.SDK/Features/UX/Prefabs, or create your own prefab and add a [PokePointer](Pointers.md#pokepointer).
+
+1.  Pick a object that you want to raise pointer events. Any ancestor of that object will be able to receive pointer events. For example, you can add a cube to the scene.
```

Comment:
MRTK Docs guide also say to avoid uses of "You":
https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/DocumentationGuide.html#writing-style

Although very ironically, the documentation guide is full of uses of "you"  :$

Reply:
Updated to remove uses of "you"